rbac/management/principal/view.py

```diff
@@ -116,13 +123,90 @@ def get(self, request):
             previous_offset = offset - limit
 
         # Attempt validating and obtaining the "prinicpal type" query
-        # parameter. For now, specifying the query parameter results
-        # in returning the mocked response.
-        principalType = validate_and_get_key(
+        # parameter.
+        pricipal_type = validate_and_get_key(
```

Comment:
Minor typo here and refs below 😄 
```suggestion
        principal_type = validate_and_get_key(
```

Reply:
Oh god lol. Thanks for pointing it out :joy: